version.txt

```diff
@@ -0,0 +1 @@
+0.4.4-add_cpm-7f59f0f8
```

Comment:
This file should not be version controled. We probably have to add this to .gitignore to not accidentally get it added :)